Makefile

```diff
@@ -47,3 +52,14 @@ docker-serve:
 test-examples:
 	scripts/test_examples.sh install
 	scripts/test_examples.sh run
+
+link-checker-setup:
+	curl https://raw.githubusercontent.com/wjdp/htmltest/master/godownloader.sh | bash
```

Comment:
I'm dubious about this (it adds an information security risk). Is it feasible to run the link checker using a container, to help mitigate that?

Reply:
@sftim https://hub.docker.com/r/wjdp/htmltest this exists, doing some investigation now.